client/interview-controller.js

```diff
@@ -0,0 +1,79 @@
+// Description:
+//   A Hubot script which helps hrs to control interviews.
+//
+// Commands:
+//  begin group Huntflow Reloaded
+//    begin admin
+//      hubot удалить интервью - remove an interview from the list
+//    end admin
+//  end group
+//
+
+const routines = require('hubot-routines')
+const utils = require('./utils')
+
+const interviewsService = require('./interviews-service')
+
+const deleteCandidateRegExp = new RegExp(/(удалить интервью кандидата)\s(((([А-Яа-я]+)|([A-Za-z]+))\s?){2})/i)
+
+module.exports = async (robot) => {
+  // Checking if server user email & password was specified
+  if (!utils.SERVER_USER_EMAIL || !utils.SERVER_USER_PASSWORD) {
+    routines.rave(robot, utils.MSG_AUTHORIZATION_DATA_NEEDED)
+    return
+  }
+
+  try {
+    await interviewsService.getAccessToken()
+  } catch (error) {
+    routines.rave(robot, utils.MSG_AUTHORIZATION_FAILED)
+    return
+  }
+
+  robot.respond(/(удалить интервью)$/i, async (msg) => {
+    if (!await routines.isAdmin(robot, msg.message.user.name.toString())) {
+      msg.send(utils.MSG_PERMISSION_DENIED)
+      return
+    }
+
+    let message = utils.MSG_ERROR_TRY_AGAIN
+
+    try {
+      const response = await interviewsService.getCandidatesList()
+      const candidates = response.data.users
+      message = candidates.length === 0 ? utils.MSG_NOT_ANY_SCHEDULED_INTERVIEWS : buildCandidatesButtons(candidates)
+    } catch (error) {
+      routines.rave(robot, error.message)
+    }
+
+    return msg.send(message)
+  })
+
+  robot.respond(deleteCandidateRegExp, async (msg) => {
+    let message = utils.MSG_ERROR_TRY_AGAIN
+
+    try {
+      const text = msg.match[0]
+      const candidate = text.split(' ').splice(4) // get elements after word 'кандидата'
+      await interviewsService.deleteCandidateInterview({ first_name: candidate[0], last_name: candidate[1] })
```

Comment:
I can't delete interviews because there is a mix of names order: `candidate[0]` is the last name.